mapper/mapper.go

```diff
@@ -186,23 +186,8 @@ func CreateAreaTypeSelector(req *http.Request, enableCustomSort bool, basePage c
 		p.ErrorId = "area-type-error"
 	}
 
-	var selections []model.Selection
-	for _, area := range areaType {
-		selections = append(selections, model.Selection{
-			Value:       area.ID,
-			Label:       area.Label,
-			Description: area.Description,
-			TotalCount:  area.TotalCount,
-		})
-	}
+	selections := mapAreaTypesToSelection(sortAreaTypes(areaType))
 
-	sort.Slice(selections, func(i, j int) bool {
-		if enableCustomSort {
```

Comment:
You also need to remove `enableCustomSort` from the `CreateAreaTypeSelector` signature